tests/fixtures/mock.py

```diff
@@ -18,3 +18,10 @@
 # functions, therefore the patching has no effect
 mp = monkeypatch()
 mp.setattr('django.utils.functional.cached_property', property)
+
+
+class FakeJob(object):
+    id = 'FAKE_JOB_ID'
+
+
+mp.setattr('rq.get_current_job', lambda: FakeJob())
```

Comment:
We discussed this in a call and also realized we cannot use the `monkeypatch` fixture in scopes other than function.
We might wanna try to explore if we can create a fixture with `autouse=True` ([check out this example](https://pytest.org/latest/monkeypatch.html#example-preventing-requests-from-remote-operations)). If that doesn't work, we'll have to settle down with the current sort-of-hackish approach.